bbc/simorgh
# WSTEAM1-350 - Article + MediaArticle grid standardisation

Resolves WSTEAM1-350

Overall changes
======
- Testing out using CSS Grid only for the grid layout on Article and MediaArticle pages
- It used the old Grid components from ArticlePage, but they are super complex, mainly to cater for IE which we don't support anymore since React 18
- Opera Mini is a browser that doesn't support Grid, but the page will be single column on Opera Mini anyway since its mobile focused, so it shouldn't be a major problem.
- Also removes psammead breakpoint and sizing imports and uses our component standards CSS pattern for applying custom styling to components
- Removes `ArticleRelatedContent` component as it doesn't appear to be used anywhere. Likely replaced by the `RelatedContentSection` component within the ArticlePage component folder.
- Adds new `gridWidths` object that defines typical page widths

**ArticlePage:**
<img width="2011" alt="Screenshot 2023-03-23 at 15 52 31" src="https://user-images.githubusercontent.com/11633031/227260366-fb64c9a9-4bd7-45ca-8cdb-dbf70e70be8e.png">

**MediaArticlePage:**
<img width="2011" alt="Screenshot 2023-03-23 at 15 52 45" src="https://user-images.githubusercontent.com/11633031/227260439-afb4988e-8028-47f0-b950-7e27231f632c.png">

**StoryPage (CPS asset) for comparison:**
<img width="2137" alt="Screenshot 2023-03-24 at 09 27 16" src="https://user-images.githubusercontent.com/11633031/227479925-37b4609b-a115-4670-9af8-8b77442b6a08.png">

Testing
======
1. Ensure that the page renders exactly the same as before this change but with new guttering applied.
2. Ensure no changes on all supported screenreaders

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
